src/client/components/Timer/Timer.js

```diff
@@ -0,0 +1,32 @@
+import React, { useEffect, useState } from 'react';
+import PropTypes from 'prop-types';
+import './Timer.styles.css';
+
+const formatTime = (timeLeftInSecond) => {
+  let minute = Math.floor(timeLeftInSecond / 60);
+  if (minute < 10) minute = `0${minute}`;
+
+  let second = timeLeftInSecond - 60 * minute;
+  if (second < 10) second = `0${second}`;
+
+  return `${minute}:${second}`;
+};
+const Timer = ({ countdown }) => {
+  const [counter, setCounter] = useState(countdown);
+
+  useEffect(() => {
```

Comment:
Can you please explain more?

Reply:
There are no repeating intervals in functional react in this sense.  setInterval normally repeats an operation at an interval, but not here. It will not repeat the interval, it will just call its parent function(Timer) again. In functional react we create repeating intervals by recalling the Timer function, not by repeating the logic in the setInterval. 
It still works, but it confuses the reader.

Reply:
So I should use setTimeout .

Reply:
Yes I think you should

Reply:
Yes it's work widh setTimeout.